cmd/promtool/main.go

```diff
@@ -364,6 +384,42 @@ func (ls lintConfig) lintDuplicateRules() bool {
 	return ls.all || ls.duplicateRules
 }
 
+const promDefaultURL = "http://localhost:9090"
+
+// Check server status - healthy & ready.
+func CheckServerStatus(serverURL, checkEndpoint string) error {
+	if serverURL == "" {
+		serverURL = promDefaultURL
+	}
+
+	u, err := url.Parse(serverURL)
+	if err != nil {
+		return fmt.Errorf("error parsing URL: %s", serverURL)
+	}
+
+	u.Path += checkEndpoint
+
+	serverURL = u.String()
+
+	// Check Health & Readiness.
+	res, err := http.Get(serverURL)
+	if err != nil {
+		return err
+	}
+	if res.StatusCode != 200 {
```

Comment:
nitpick :upside_down_face: 
```suggestion
	if res.StatusCode != http.StatusOK {
```

Reply:
Addressed